### PR TITLE
Better explain conditions for deployment with apidev.py script docs (#1292)

### PR DESCRIPTION
**Pull request by jessebrennan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1232,9 +1232,20 @@ The script gives you a url where your current version of the documentation
 is visible. Change the docs in `azul/service/app.py`, save, and refresh the page and
 your changes will appear immediately along with any warnings or errors they introduce.
 
-The script pulls the skeleton for what it displays from the currently activated
-deployment. If you add new endpoints or change endpoint parameters, this won't be
-reflected nor validated by the script. To remedy this run
+# FIXME
+Some changes to `app.py` won't be displayed properly by this script unless the changes
+are deployed in the active deployment. Specifically, changes to
+- the Swagger spec or
```
Comment (GPelayo) [inline README.md]:
```suggestion
- the Swagger spec
```

File: README.md [hunk 2]
```diff
@@ -1232,9 +1232,17 @@ The script gives you a url where your current version of the documentation
 is visible. Change the docs in `azul/service/app.py`, save, and refresh the page and
 your changes will appear immediately along with any warnings or errors they introduce.
 
-The script pulls the skeleton for what it displays from the currently activated
-deployment. If you add new endpoints or change endpoint parameters, this won't be
-reflected nor validated by the script. To remedy this run
+Some changes to `app.py` won't be displayed properly by this script unless the changes
```
Comment (hannes-ucsc) [inline README.md]:
```suggestion
Some changes to `app.py` won't take effect with this script unless the changes
```

File: README.md [hunk 3]
```diff
@@ -1232,9 +1232,17 @@ The script gives you a url where your current version of the documentation
...
+are deployed in the active deployment. Specifically, changes to
+- the Swagger spec
```
Comment (hannes-ucsc) [inline README.md]:
```suggestion

- the Swagger spec
```

Comment (hannes-ucsc) [reply README.md]:
```suggestion
- the OpenAPI spec in either the `…` global variable or the `spec` parameter to the `route` decorator.
```

File: README.md [hunk 4]
```diff
@@ -1232,9 +1232,17 @@ The script gives you a url where your current version of the documentation
...
+- the *body* of pre-existing route functions
+
+are **fine** and **won't** require redeployment. Any other changes **will**, such as
```
Comment (hannes-ucsc) [inline README.md]:
```suggestion
will **not** require redeployment. Any other changes **will**, such as

```

Note the added newlines.

**Pull request merged**